File: javascript-exercicios/ex074.js
const prompt = require("prompt-sync")()

function randint(){
    return  Math.floor(Math.random()*10) + 1
}

let numeros = [
    randint(),randint(),
    randint(),randint(),
    randint(),
]

console.log(`os valores sorteados foram: `,numeros.join(' '))

let maior = Math.max(...numeros)
let menor = Math.min(...numeros)
console.log(`o maior valor digitado foi ${maior}`)
console.log(`o menor valor digitado foi ${menor}`)

File: curso-python/ex074.py
from random import randint

numeros = (randint(1,10),randint(1,10),
     randint(1,10),randint(1,10),
     randint(1,10))

print("os valores sorteados foram: ",end='')
for n in numeros:
    print(f"{n}",end=' ')

print()

print(f"o maior valor sorteado foi {max(numeros)}")
print(f"o menor valor sorteado foi {min(numeros)}")